### DISCUSSION TITLE
Ops... foi mal, perdi o encontro hoje, minhas desculpas!

### DISCUSSION THREAD

**Discussion by villares** [clube-de-leitura-python/pense-em-python-2021#26] [Conversa geral]
Queridas pessoas participates. Hoje eu dei uma vacilada, me confundi, achei que era semana que vem o encontro, e dormi (!) estava super cansado... Minhas desculpas. Por sorte parece que rolou ok a conversa, o @rgobatto me contou que foi um pouco confuso mas deu para conversar, espero vê-los no próximo dia 19...

Aproveito pra comentar que temos um fork do livro em: https://github.com/clube-de-leitura-python/PensePython2e
para o qual podemos propor comentários, anotações e até exercícios alternativos!

**Comment by chicortiz** [comment]
Tranquilo Alê! Sentimos sua falta. O Yasuda fez um bonito discurso filosófico.
